apps/core/lib/core/clients/hydra.ex

```diff
@@ -127,6 +127,8 @@ defmodule Core.Clients.Hydra do
 
   defp handle_response({:ok, %{status_code: code, body: body}}, type) when code in 200..299,
     do: {:ok, Poison.decode!(body, as: type)}
+  defp handle_response({:ok, %{status_code: code, body: body}}, _type) when code in 400..499,
```

Comment:
For `accept_login`, if the response is `410` it will contain a `redirect_to` which should probably be returned by our API so the user is redirected properly.

Reply:
this code does handle the 409 response already, it'll fall back to the clause below and return an `{:error, :unauthorized}` tuple which will bubble to a gql error

Reply:
It'll also log the error (why we're seeing them in the logs in all likelihood)